### DISCUSSION TITLE
Image | Orangepi 5 pro

### DISCUSSION THREAD

**Discussion by Seb-tech77** [MichaIng/DietPi#7273] [Image Request] [answered]
Hello, would it be possible to add orangepi 5 pro to dietpi? 
I would like to use it as a server but as the card is not officially supported by armbian, I'm having trouble finding a working image.
Sincerely 

Armbian: https://www.armbian.com/orange-pi-5-pro/
Orangepi: https://drive.google.com/drive/folders/1KnmBQ3Z0M_5snRC24LjhKb8_tKbcfOkw

Technical Specifications:
Processor (SoC): Rockchip RK3588S:
4x Cortex-A76 cores (2.4 GHz) and 4x Cortex-A55 cores (1.8 GHz).
GPU: Mali-G610 MP4 with support for OpenGL ES 3.2, OpenCL 2.2, and Vulkan 1.2.
Integrated NPU offering AI processing power up to 6 TOPS.
RAM: LPDDR5, available in 4GB, 8GB, or 16GB configurations.
Storage:
eMMC socket.
SPI NOR Flash support.
M.2 slot for NVMe (PCIe 2.0 x1) or SATA3 SSDs.
MicroSD card slot.
Video Output:
HDMI 2.1 supporting 8K at 60Hz.
HDMI 2.0 for 4K at 60Hz (via DisplayPort-to-HDMI bridge).
MIPI DSI connector (4 lanes) for 4Kp60 displays.
Audio:
ES8388 audio codec.
3.5mm jack for headphones or microphone.
Built-in microphone and HDMI eARC support.
Networking:
Gigabit Ethernet with PoE option.
Dual-band Wi-Fi 5 and Bluetooth 5.0 with BLE.
Connectivity:
USB ports: 1x USB 3.0, 3x USB 2.0, and 2x USB 2.0 via headers.
40-pin GPIO header supporting UART, PWM, I2C, SPI, CAN, and GPIO.
Dimensions: 89 x 56 mm, weighing 62 grams.

**Comment by Joulinar** [comment, marked as answer]
pls check https://github.com/MichaIng/DietPi/issues/7255

**Comment by MichaIng** [comment]
This should make the fact clear 🙂 https://github.com/MichaIng/DietPi-Website/commit/e8d8804
